src/components/OverlayManager/Password/index.tsx

```diff
@@ -21,7 +21,7 @@ const Password: React.FC<{ overlay: OverlayContextInterface }> = ({
     <div className="password-reset">
       <Online>
         <div className="overlay__header">
-          <p>Reset your password</p>
+          <div className="overlay__header-text">Reset your password</div>
```

Comment:
Div should not have directly text in it.
Also please clean commits.

Reply:
@piotrgrundas in many components `Div` have directly text in it . Should i change them all?